spec/services/email_sender/notify_sender_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+require "notifications/client"
+
+RSpec.describe EmailSender::NotifySender do
+  describe "#call" do
+    it "requires an address" do
```

Comment:
Do we need to test this now it's a keyword argument? We wouldn't test that a method with a 'normal' argument didn't work if you didn't supply any.

Reply:
quite right - fixed f6c9a48650302f23b73623ad9318b3165e63cdfb